### PR TITLE
Feature/269 task 273 add or remove padding

### PR DESCRIPTION
**Pull request by ManveerPandher**
What has been done:
* * Aligned the Horizontal line with Produver Count, Gas Injector Count etc all the way to the left to be aligned with the rest
* On the view for Case, Added spacing between the horizontal line with Producer Count, Gas Injector Count etc, and the CaseAsset underneath

Azure Task: 
* https://dev.azure.com/2S-IAF/DCD/_workitems/edit/273

### CODE REVIEW

File: frontend/src/Components/NumberInput.tsx [hunk 1]
```diff
@@ -35,7 +35,7 @@ const NumberInput = ({
     }
 
     return (
-        <WrapperColumn style={{ paddingLeft: 10 }}>
```
Comment (DanielBohme) [inline frontend/src/Components/NumberInput.tsx:38]: Please do the same on NumberInputInherited as well

File: frontend/src/Components/CaseAsset.tsx [hunk 2]
```diff
@@ -10,6 +10,7 @@ import { EMPTY_GUID } from "../Utils/constants"
 import { unwrapCase } from "../Utils/common"
 
 const Wrapper = styled.div`
+    margin-top: 1rem;
```
Comment (DanielBohme) [inline frontend/src/Components/CaseAsset.tsx:13]: Add "margin-bottom: -1rem;" as well to make the space between assetlinks a bit smaller? The extra top margin creates a lot of space between assetlinks

File: frontend/src/Components/NumberInput.tsx (hunk 1 above)
Comment (ManveerPandher) [reply frontend/src/Components/NumberInput.tsx:38]: Done

**Comment by ManveerPandher** [comment]
Resolved the comments which @DanielBohme requested

**Review by DanielBohme — APPROVED** [review]
LGTM

**Pull request merged**